pxc-80-backup/recovery-pvc-joiner.sh

```diff
@@ -30,19 +33,14 @@ function check_ssl() {
     fi
 }
 
-parse_sst_info() {
-   local source_path=$1
-   awk -F "=" '/transition-key/ {print $2}' "$source_path"
-}
-
 check_ssl
 ping -c1 $RESTORE_SRC_SERVICE || :
 rm -rf /datadir/*
 
-socat -u "$SOCAT_OPTS" stdio | xbstream -x -C /datadir --parallel=$(grep -c processor /proc/cpuinfo)
+socat -u "$SOCAT_OPTS" stdio >/datadir/sst_info
 socat -u "$SOCAT_OPTS" stdio | xbstream -x -C /datadir --parallel=$(grep -c processor /proc/cpuinfo)
 
-transition_key=$(parse_sst_info "/datadir/sst_info")
+transition_key=$(vault_get /datadir/sst_info)
 if [[ -n $transition_key ]]; then
     encrypt_prepare_options="--transition-key=\$transition_key"
 fi
```

Comment:
hide transition-key from logs
```suggestion
set +o xtrace
transition_key=$(vault_get /datadir/sst_info)
if [[ -n $transition_key ]]; then
    encrypt_prepare_options="--transition-key=\$transition_key"
    echo transition-key exists
fi
echo + xtrabackup ${XB_USE_MEMORY+--use-memory=$XB_USE_MEMORY} --prepare --binlog-info=ON --rollback-prepared-trx --xtrabackup-plugin-dir=/usr/lib64/xtrabackup/plugin --target-dir=/datadir
```